### PR TITLE
Refactors 2016 Agenda

### PR DESCRIPTION
**Pull request by antonjb**
Refactors the 2016 Agenda to match the modern layout and structure.

### CODE REVIEW

File: pages/agenda/2019.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const agendaParams: AgendaPageParameters = {
 }
 
 const Agenda2019: React.FC<AgendaPageParameters & { sessions: Session[] }> = ({ sessions }) => {
-  const date = moment.parseZone('2019-06-23T17:00:00+08:00')
+  const date = moment.parseZone('2019-06-23T08:00:00+08:00')
```
Comment (antonjb) [inline pages/agenda/2019.tsx]: Not sure what happened here but the time was wrong for 2019

File: pages/agenda/2017.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ const Agenda2017: React.FC<AgendaPageProps & { sessions: Session[] }> = ({ sessi
               sponsors={From2019.Sponsors}
```
Comment (mzaatar) [inline pages/agenda/2017.tsx]: Is this alright? `From2019. Sponsors`

Comment (antonjb) [reply pages/agenda/2017.tsx]: whoops. That's very not ok. Nice catch!

File: pages/agenda/2016.tsx [hunk 3]
```diff
@@ -1,211 +1,225 @@
...
+        acceptingFeedback={false}
+        selectedSessionId={undefined}
+        hideLevel={true}
+        hideTags={true}
+        render={(agendaSessions, _, onSelect) => {
+          return (
+            <AgendaProvider
+              onSelect={onSelect}
+              sessions={agendaSessions}
+              sponsors={From2019.Sponsors}
```
Comment (mzaatar) [inline pages/agenda/2016.tsx]: Same here

File: pages/agenda/2017.tsx (hunk 2 above)
Comment (mzaatar) [reply pages/agenda/2017.tsx]: Only is they are the same sponsors, but I doubt

File: pages/agenda/2016.tsx (hunk 3 above)
Comment (antonjb) [reply pages/agenda/2016.tsx]: 🤦‍♂ 

Have fixed both. Thanks for catching

**Pull request merged**